app/layout.tsx

```diff
@@ -14,15 +17,16 @@ export const metadata: Metadata = {
 export default function RootLayout({
   children,
 }: Readonly<{
-  // eslint-disable-next-line no-undef
   children: React.ReactNode;
 }>) {
   return (
     <html lang="en">
       <body className={roboto.className}>
         <main>
+          <Navigation />
           <Suspense fallback={<Loading />}>{children}</Suspense>
         </main>
+        <Footer />
```

Comment:
The navigation should be placed outside of main.
Read more about main tag here: https://www.w3schools.com/tags/tag_main.asp